pkg/core/state/storage_item.go

```diff
@@ -6,18 +6,15 @@ import (
 
 // StorageItem is the value to be stored with read-only flag.
 type StorageItem struct {
-	Value   []byte
-	IsConst bool
+	Value []byte
```

Comment:
We can drop `struct` wrapper altogether leaving it as just a slice alias.

Reply:
Actually now there is no need to use var-encoding, we can serialize items directly. Need to agree with upstream though.

Reply:
For some reason I thought it already is the case (no var bytes), but it's not, so let's try to fix that in neo-project/neo#2379.

Reply:
It's merged now, so we can simplify things further.

Reply:
Fixed.